lisp/forge-commands.el

```diff
@@ -55,6 +55,8 @@ for a repository using the command `forge-add-pullreq-refspec'."
     ("l p" "pull-requests" forge-list-pullreqs)
     ("l n" "notifications" forge-list-notifications)
     ("l r" "repositories"  forge-list-repositories)
+    ("m p" "authored pull-requests" forge-list-authored-pullreqs)
+    ("m i" "authored issues"        forge-list-authored-issues)
     (7 "l a" "awaiting review"     forge-list-requested-reviews)
```

Comment:
@tarsius What's the `7` here for? I couldn't find anything in the docs that told me.

Reply:
This is the relevant documentation: https://magit.vc/manual/transient/Enabling-and-Disabling-Suffixes.html#Enabling-and-Disabling-Suffixes

The gist of it is that each command has a level, a number from 1 to 7, associated with it. If the the user's transient-default-level value (which is 4 by default) is greater than or equal to the command's level then it is shown in the transient.

Reply:
Great - thanks!